defaults/main.yml

```diff
@@ -3,7 +3,8 @@
 wimpy_project_name: "{{ wimpy_project | regex_replace('[^a-zA-Z0-9]', '-') }}"
 wimpy_github_project_url: ""
 
-wimpy_docker_image_name: "{{ wimpy_project_name }}" # Name of the docker image that will be created
+wimpy_docker_registry: "{{ wimpy_aws_account_id ~ '.dkr.ecr.' ~ wimpy_aws_region ~ '.amazonaws.com/' if wimpy_aws_account_id is defined else '' }}"
+wimpy_docker_image_name: "{{ wimpy_docker_registry }}{{ wimpy_project_name }}" # Name of the docker image that will be created
```

Comment:
Exactly. Althought keep in mind that `wimpy_aws_account_id` is only needed to login to AWS ECR. You only set it if you use ECR.

Anyway, I don't really like it either. How can I make it better?